shelley/chain-and-ledger/executable-spec/src/STS/Chain.hs

```diff
@@ -67,8 +68,10 @@ instance
   type Environment (CHAIN crypto) = Slot
 
   data PredicateFailure (CHAIN crypto)
-    = BbodyFailure (PredicateFailure (BBODY crypto))
-    | BheadFailure (PredicateFailure (BHEAD crypto))
+    = HeaderSizeTooLargeTICK
```

Comment:
Why the postfix "TICK" here, given these are no longer in the `TICK` rule

Reply:
it shouldn't be, good catch!